python/cuml/test/test_random_forest.py

```diff
@@ -751,7 +751,7 @@ def test_rf_printing(capfd, n_estimators, detailed_printing):
     # Initialize cuML Random Forest classification model
     cuml_model = curfc(handle=handle, max_features=1.0, rows_sample=1.0,
                        n_bins=16, split_algo=0, split_criterion=0,
-                       min_rows_per_node=2, random_state=23707, n_streams=1,
+                       min_samples_leaf=2, random_state=23707, n_streams=1,
```

Comment:
In the next test PR, it would be helpful to have direct tests of min_samples_leaf and min_samples_split - setting them to a high level then parsing the json dumps to check they are not violated.

Reply:
@JohnZed This PR adds a unit test for `min_samples_leaf` and `min_samples_split`. Do you mean that we should also add an end-to-end test (in Python)?